### ISSUE TITLE
[Severe] Controls and media elements should have labels (1)

### ISSUE THREAD

**Issue by SalwaAlnazly** [FACG2/Inventory-Management-System#97]
https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_text_01

`<input type="password" class="txt" id="signup-password-confirm" name="confirmPassword" placeholder="أدخل كلمة المرور" required="">`
